### PR TITLE
=htc #440 Exclude port when rendering X-Forwarded-For and X-Real-Ip

### PR DESCRIPTION
**Pull request by jonas**
Refs #440.

### CODE REVIEW

File: akka-http-core/src/main/scala/akka/http/scaladsl/model/RemoteAddress.scala [hunk 1]
```diff
@@ -56,6 +56,13 @@ object RemoteAddress {
     try IP(InetAddress.getByAddress(bytes)) catch { case _: UnknownHostException ⇒ Unknown }
   }
 
+  private[akka] val renderWithoutPort = new Renderer[RemoteAddress] {
+    def render[R <: Rendering](r: R, address: RemoteAddress): r.type = address match {
+      case IP(ip, _) ⇒ r ~~ ip.getHostAddress
+      case _         ⇒ r ~~ address
```
Comment (jlprat) [inline akka-http-core/src/main/scala/akka/http/scaladsl/model/RemoteAddress.scala]: Nice alignment!

File: akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala [hunk 2]
```diff
@@ -953,6 +953,7 @@ final case class `WWW-Authenticate`(challenges: immutable.Seq[HttpChallenge]) ex
 // http://en.wikipedia.org/wiki/X-Forwarded-For
 object `X-Forwarded-For` extends ModeledCompanion[`X-Forwarded-For`] {
   def apply(first: RemoteAddress, more: RemoteAddress*): `X-Forwarded-For` = apply(immutable.Seq(first +: more: _*))
+  implicit val addressRenderer = RemoteAddress.renderWithoutPort
```
Comment (jlprat) [inline akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala]: Naming is complicated, but maybe some other name not so easy to be confused with the other implicit?

Comment (jonas) [inline akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala]: What about `singleAddressRenderer` or `remoteAddressRenderer`?

Comment (jlprat) [inline akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala]: I think I prefer `singleAddressRenderer` :smiley:

Comment (jonas) [inline akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala]: OK, done

**Comment by jonas** [comment]
PLS BUILD

**Pull request merged**